homeassistant/components/fints/sensor.py

```diff
@@ -173,6 +176,7 @@ def __init__(self, client: FinTsClient, account, name: str) -> None:
         """Initialize a FinTs balance account."""
         self._client = client
         self._account = account
+        self._attr_unique_id = account.iban
```

Comment:
This is not a fix, but a new feature. Please separate those into different PRs

Reply:
Removed. This will be re-implemented in another potential PR.